src/main/java/JsoupBadLinks.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import java.net.*;
+import java.io.*;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class JsoupBadLinks {
+    private static final int BAD_CODE = 404;
+    private static final int BAD_CODE2 = 505;
```

Comment:
@dwaynewade0033 As I understood it is necessary to check on all SERVER_ERROR (50x), and not only 505